### PR TITLE
RUN: Uses --exact flag when generating run configuration for a single…

### PR DESCRIPTION
**Pull request by pasa**

### CODE REVIEW

File: src/main/kotlin/org/rust/cargo/runconfig/command/CargoCommandConfiguration.kt [hunk 1]
```diff
@@ -43,6 +43,7 @@ class CargoCommandConfiguration(
     var channel: RustChannel = RustChannel.DEFAULT
     var command: String = "run"
     var nocapture: Boolean = true
+    var exact: Boolean = false
```
Comment (matklad) [inline src/main/kotlin/org/rust/cargo/runconfig/command/CargoCommandConfiguration.kt]: I think it wold be better not to surface this flag up to `CargoCommandConfiguration`, and instead just add it as an `additionalArgument`: we try to keep the number for various checkboxes for Cargo run configuration as small as possible :)

File: src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestRunConfigurationProducer.kt [hunk 2]
```diff
@@ -83,12 +83,14 @@ class TestConfig(
     val sourceElement: RsCompositeElement,
     val configurationName: String,
     testPath: String,
-    target: CargoWorkspace.Target
+    target: CargoWorkspace.Target,
+    exact: Boolean
 ) {
     val cargoCommandLine: CargoCommandLine = CargoCommandLine(
         "test",
         target.cargoArgumentSpeck + testPath,
```
[the remark this answers is not in the document]
Comment (pasa) [reply src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestRunConfigurationProducer.kt]: Ok. I 've made it in this way. It looks clumsy a little to my mind. I mean we are managing cargo test additional parameters --nocapture and --exact in different places.  May be we should add some list of additional arguments in CargoCommandLine class which will be placed strictly after  "--"?

**Comment by matklad** [comment]
LGTM, thanks @pasa !

Have you singled the electronic CLA yet (https://github.com/intellij-rust/intellij-rust/blob/master/CONTRIBUTING.md#cla)? If yes, could you add your GitHub username to https://github.com/intellij-rust/intellij-rust/blob/master/CONTRIBUTORS.txt?

File: src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestRunConfigurationProducer.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (matklad) [reply src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestRunConfigurationProducer.kt]: Interestingly, https://github.com/intellij-rust/intellij-rust/pull/1862 touches on the same issues. I think we should first merge both pull request and then refactor code a bit!

**Comment by pasa** [comment]
Yep. I 've filled this form but do not know what should happen next. I didn't receive any response email or something. I 've updated CONTRIBUTORS recently.

**Comment by matklad** [comment]
bors: r+

Thanks!

File: src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestRunConfigurationProducer.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (matklad) [reply src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestRunConfigurationProducer.kt]: Ok, I've tried to unify code from both PRs in https://github.com/intellij-rust/intellij-rust/pull/1882

**Pull request merged**